samples/trace/package.json

```diff
@@ -11,9 +11,9 @@
   },
   "repository": "GoogleCloudPlatform/opentelemetry-operations-js",
   "dependencies": {
-    "@google-cloud/opentelemetry-cloud-trace-exporter": "0.3.0",
-    "@opentelemetry/api": "0.7.0",
-    "@opentelemetry/tracing": "0.7.0",
-    "@opentelemetry/node": "0.7.0"
+    "@google-cloud/opentelemetry-cloud-trace-exporter": "file:../../packages/opentelemetry-cloud-trace-exporter",
```

Comment:
Is this intentional ?

Reply:
Yes, so you can run the sample against the repo's version. I can change it back, but then it's hard to verify the sample works with the current commit.